lib/matplotlib/backends/backend_pdf.py

```diff
@@ -482,21 +511,21 @@ def __init__(self, filename, metadata=None):
                          if v is not None}
 
         self.fontNames = {}     # maps filenames to internal font names
-        self.nextFont = 1       # next free internal font name
+        self.internalFontName = (Name(f'F{i}') for i in itertools.count(1))
```

Comment:
I don't really mind the API break (it's clearly intended to be a private API), but please add an API change note and make the new APIs private.